From 130bc21708c2b0a445688ed3e6be68dd6b34b450 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Fri, 31 Oct 2025 13:02:32 +0200
Subject: [PATCH 1/1] platform/marvell-prest: ramdisk compression none arm64

According to boot startup the ramdisk compression is deprecated.
Config sonic_fit.its with compression "none".

Signed-off-by: Yan Markman <ymarkman@marvell.com>
---
 platform/marvell-prestera/sonic_fit.its | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/platform/marvell-prestera/sonic_fit.its b/platform/marvell-prestera/sonic_fit.its
index 242232cfe..4beb22566 100644
--- a/platform/marvell-prestera/sonic_fit.its
+++ b/platform/marvell-prestera/sonic_fit.its
@@ -46,7 +46,7 @@
             type = "ramdisk";
             arch = "arm64";
             os = "linux";
-            compression = "gzip";
+            compression = "none";
             load = <0x2 0x6000000>;
             entry = <0x2 0x6000000>;
             hash@1 {
@@ -72,7 +72,7 @@
             type = "ramdisk";
             arch = "arm64";
             os = "linux";
-            compression = "gzip";
+            compression = "none";
             load = <0x6000000>;
             entry = <0x6000000>;
             hash@1 {
@@ -98,7 +98,7 @@
             type = "ramdisk";
             arch = "arm64";
             os = "linux";
-            compression = "gzip";
+            compression = "none";
             load = <0x18000000>;
             entry = <0x18000000>;
             hash@1 {
-- 
2.34.1

